packages/grid/_modules_/grid/constants/eventsConstants.ts

```diff
@@ -305,4 +305,8 @@ export enum GridEvents {
    * Fired when a column visibility changes. Called with a [[GridColumnVisibilityChangeParams]] object.
    */
   columnVisibilityChange = 'columnVisibilityChange',
+  /**
+   * Fired when the rows in the viewport change. Called with a [[GridViewportRowsChange]] object.
+   */
+  gridViewportRowsChange = 'viewportRowsChange',
```

Comment:
We don't use the `grid` prefix anymore.

```suggestion
  viewportRowsChange = 'viewportRowsChange',
```